package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
 	"name": "execute-code",
-	"version": "0.18.0",
+	"version": "1.4.0",
```

Comment:
Version bumps are done by @twibiral on the `master` branch, so I think that this commit might not be in accordance with that? Not 100% sure :)

Reply:
@alfu32 Please remove this change. The `package-lock.json` must never be touched. The version number is automatically updated by a script when I create a release.

@chlohal Thanks for noticing this